api/log.go

```diff
@@ -35,6 +35,8 @@ func (h *LogHandlers) LogHandler(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+type DatasetLogItem = base.DatasetLogItem
```

Comment:
This needs a comment describing it, which begins `// DatasetLogItem ...` in order to pass our `golint` step. For example `// DatasetLogItem aliases the type from base`.